content/5-advanced-usages/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Miscellaneous
-weight: 70
-pre: "<b>7. </b>"
+title: Advanced Usages
```

Comment:
```suggestion
title: Advanced Topics
```

Reply:
Fixed in https://github.com/coreruleset/documentation/pull/137/commits/53a0433399d443048850bf3b8de34a4eed9b087b